core/mondoo-linux-security.mql.yaml

```diff
@@ -1080,19 +1081,16 @@ queries:
         ```
   - uid: mondoo-linux-security-auditing-for-processes-that-start-prior-to-auditd-is-enabled
     title: Ensure auditing for processes that start prior to auditd is enabled
-    impact: 80
+    impact: 50
     mql: |
-      if( file("/boot/grub2/grub.cfg" ).exists) {
-        file("/boot/grub2/grub.cfg").content.lines.where( _ == /^[^#]/ ).any(_ == /audit(\s+)?\=(\s+)?1/)
+      if(file("/boot/grub2/grub.cfg").exists) {
+        file("/boot/grub2/grub.cfg").content.contains("audit=1")
```

Comment:
maybe we can use something like:

```
file("/boot/grub2/grub.cfg").content == /audit\s=\s1/
```

Reply:
Is it intentional that we do not ignore the comments anymore?

Reply:
sorry my mistake

Reply:
no, thanks for the hint